src/css/styles.css

```diff
@@ -65,7 +65,8 @@ body {
   justify-content: space-between;
 }
 
-form > label {
+/* //check to maek sure this works still for main: */
```

Comment:
There's quite a few commented out pieces of code in this file to delete whenever you have a moment!